lib/tasks/dev.rake

```diff
@@ -1,3 +1,71 @@
 desc "Fill the database tables with some sample data"
 task({ :sample_data => :environment }) do
+  if Rails.env.development?
+    FollowRequest.destroy_all
+    Comment.destroy_all
+    Like.destroy_all
+    Photo.destroy_all
+    User.destroy_all
+  end
+
+  12.times do
+    name = Faker::Name.first_name
+    u = User.create(
+      email: "#{name}@example.com",
+      password: "password",
+      username: name,
+      private: [true, false].sample
+    )
+
+    p u.errors.full_messages
```

Comment:
errors.full_messages is a good method to see right away if anything goes wrong. Good job!